### PR TITLE
[C-3018][C-2965] fix upload album/playlist description field styling

### PR DESCRIPTION
**Pull request by amendelsohn**
### Description

- Fixes description limit location
- Migrate to plain button

### How Has This Been Tested?
Local web

### Screenshots

Screenshot 2023-08-28 at 4 26 24 PM
Screenshot 2023-08-28 at 4 26 09 PM
Screenshot 2023-08-28 at 4 34 12 PM

### CODE REVIEW

File: packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css [hunk 1]
```diff
@@ -43,6 +43,10 @@
   gap: var(--unit-12);
 }
 
+.actionButton {
```
Comment (dylanjeffers) [inline packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css:46]: can this be built into the plain button? if we are going to supported subdued text on plain button i feel like should be a prop?

File: packages/web/src/pages/upload-page/forms/EditTrackForm.tsx [hunk 2]
```diff
@@ -180,18 +181,16 @@ const MultiTrackFooter = () => {
...
-        variant={HarmonyButtonType.PLAIN}
         text={messages.prev}
         iconLeft={IconCaretLeft}
         onClick={goPrev}
         disabled={prevDisabled}
         type='button'
       />
-      <HarmonyButton
+      <HarmonyPlainButton
         className={cn({ [styles.disabled]: nextDisabled })}
```
Comment (dylanjeffers) [inline packages/web/src/pages/upload-page/forms/EditTrackForm.tsx:193]: kinda the same here, like we should just have a disabled prop. if we just want to do follow-ons to the butons when we do harmony, mind just adding a ticket so we have it covered?

File: packages/web/src/components/data-entry/TextAreaV2.module.css [hunk 3]
```diff
@@ -57,6 +57,7 @@
 .scrollArea {
   display: flex;
   flex-direction: row;
+  flex-grow: 1;
```
Comment (dylanjeffers) [inline packages/web/src/components/data-entry/TextAreaV2.module.css:60]: nice

**Review by dylanjeffers — APPROVED** [review]
excellent

File: packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css (hunk 1 above)
Comment (amendelsohn) [reply packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css:46]: thoughts on subdued vs a color prop?

Comment (amendelsohn) [reply packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css:46]: wait we totally have a subdued variant. nvm

Comment (dylanjeffers) [reply packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css:46]: hell yeah

Comment (amendelsohn) [reply packages/web/src/pages/upload-page/fields/CollectionTrackField.module.css:46]: man having these standard components is going to be so amazing

**Pull request merged**